CONTRIBUTE.md

```diff
@@ -22,8 +22,15 @@ If you encounter permissions errors you might want to `chown -R www-data:www-dat
 
 ### Run unit tests
 
+#### First run
+
+On first run you might need to setup the testing sqlite database first.
+Run the command `php artisan migrate --database sqlite-testing` to setup the database.
```

Comment:
No it won't, I tested this yesterday and you just get sql errors or something iirc. So we need to migrate the database, and I think it's best to use the laravel command for that instead of manual touching

Reply:
Hmm. Are you sure? I just tested the same locally and it worked. All I needed to do was to create the database file.

Reply:
Updated comment with the alternate suggestion, although I haven't been able to reproduce issues with testing with only `touch database/testing.sqlite` yet.

Reply:
When I ran test yesterday it was without any database file, so it just errored. Though if you mean that touching the file and then running tests, then it's possible it works and auto-migrates.

Though I'm more fan of using the artisan commands to accomplish the same thing, because suddenly Laravel does a change in their test handling and then it's best to use the "standard way" of testing, rather than manually touching files imo.